introduction.md

```diff
@@ -1,2 +1,8 @@
 Hello, new students of the dojo. Please learn the dojo rules, and lets start the training.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?